packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.tsx

```diff
@@ -0,0 +1,137 @@
+    defaultMessage: '{progress} steps complete',
+    values: {
+      progress: `${numberCompleteSteps}/${numberSteps}`,
+    },
+  });
+};
+
+export type GuideCardSolutions = 'search' | 'observability' | 'security';
+
+export interface GuideCardsProps {
+  isLoading: boolean;
+  activateGuide: (guideId: GuideId, guideState?: GuideState) => Promise<void>;
+  navigateToApp: ApplicationStart['navigateToApp'];
+  activeFilter: GuideFilterValues;
+  guidesState: GuideState[];
+}
+export const GuideCards = ({
+  isLoading,
+  activateGuide,
+  navigateToApp,
+  activeFilter,
+  guidesState,
+}: GuideCardsProps) => {
+  return (
+    <EuiFlexGroup wrap responsive justifyContent="center">
+      {guideCards.map((card, index) => {
+        let guideState: GuideState | undefined;
+        if (card.guideId) {
+          guideState = guidesState.find((state) => state.guideId === card.guideId);
+        }
+        const onClick = async () => {
+          if (card.guideId) {
+            await activateGuide(card.guideId, guideState);
+          } else if (card.navigateTo) {
+            await navigateToApp(card.navigateTo?.appId, {
+              path: card.navigateTo.path,
+            });
+          }
+        };
+        const isHighlighted = activeFilter === 'all' || activeFilter === card.solution;
+        const isComplete = guideState && guideState.status === 'complete';
+        const progress = isComplete ? undefined : getProgressLabel(guideState);
+        return (
+          <EuiFlexItem key={index} grow={false}>
+            <EuiCard
+              isDisabled={isLoading}
+              onClick={onClick}
+              css={cardCss}
+              display={isHighlighted ? undefined : 'transparent'}
+              hasBorder={!isHighlighted}
+              title={
+                <>
+                  <EuiSpacer size="s" />
+                  <h3 style={{ fontWeight: 600 }}>{card.title}</h3>
+                </>
+              }
+              titleSize="xs"
+              betaBadgeProps={{
+                label: card.solution,
+              }}
+              description={
+                <>
+                  {progress && (
+                    <EuiTextColor color="subdued">
+                      <small>{progress}</small>
+                    </EuiTextColor>
+                  )}
+                  {isComplete && (
+                    <EuiFlexGroup gutterSize="s" alignItems="center">
+                      <EuiFlexItem grow={false}>
+                        <EuiIcon type="checkInCircleFilled" color="success" />
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <small>
+                          {i18n.translate(
+                            'guidedOnboardingPackage.gettingStarted.cards.completeLabel',
+                            {
+                              defaultMessage: 'Guide Complete',
```

Comment:
```suggestion
                              defaultMessage: 'Guide complete',
```
```suggestion
                              defaultMessage: 'Guide Complete',
```

Reply:
What about changing "1/2 steps complete" to "1 of 2 steps complete".  Its a little longer but might convey the message better. @cindychangy any thoughts?

Reply:
@gchaps, I think that makes sense and you are right it's probably better to be more concise, let's change this to use "of" instead of the "/"